packages/cli/src/lib/strings.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+type Messages = {
+  promptHostMessage: string;
+  promptName: string;
+  promptLauncherName: string;
+  promptDisplayMode: string;
+  promptThemeColor: string;
+  promptBackgroundColor: string;
+  promptStartUrl: string;
+  promptIconUrl: string;
+  promptMaskableIconUrl: string;
+  promptMonochromeIconUrl: string;
+  promptShortcuts: string;
+  promptPackageId: string;
+  promptKeyPath: string;
+  promptKeyAlias: string;
+  promptCreateKey: (keypath: string) => string;
+  promptKeyFullName: string;
+  promptKeyOrganizationalUnit: string;
+  promptKeyOrganization: string;
+  promptKeyCountry: string;
+  promptKeystorePassword: string;
+  promptKeyPassword: string;
+  promptNewAppVersionName: string;
+  errorMaxLength: (maxLength: number, actualLength: number) => string;
+  errorMinLength: (minLength: number, actualLength: number) => string;
+  errorRequireHttps: string;
+  errorInvalidUrl: (url: string) => string;
+  errorInvalidColor: (color: string) => string;
+  errorInvalidDisplayMode: (displayMode: string) => string;
+}
+
+export const en: Messages = {
```

Comment:
If you make this `en_US` I could create an `en_GB` where we actually spell colour correctly.

Reply:
done. We don't have internationali**s**ation yet, but this is a step in that direction.